checkov/secrets/runner.py

```diff
@@ -50,6 +50,12 @@
 
 ENTROPY_KEYWORD_LIMIT = 3
 PROHIBITED_FILES = ['Pipfile.lock', 'yarn.lock', 'package-lock.json', 'requirements.txt']
+ADDED_TO_SECRET_SCAN_FILES_NAMES = ['settings.py', 'main.py', 'application.py', 'config.py', 'app.js', 'config.js', 'dev.js',
+                              'default.json', 'config.json', 'appsettings.json', 'credentials.json', 'db.properties',
+                              'application.properties', 'private.pem', 'privtaekey.pem', 'ndex.php', 'config.php',
+                              'config.xml', 'strings.xml', 'config.yml', 'travis.yml', 'docker-compose.yml',
+                              'secrets.yml', 'app.module.ts', 'environment.ts', 'Dockerfile']
```

Comment:
please check the regex we use for Dockerfiles and leverage it to also scan files like `Dockerfile.local` etc.